CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 4.0.0-alpha.1
+
+#### General
+
+* `FEAT`: new loading indicator ([#1719](https://github.com/camunda/camunda-modeler/issues/1719))
+* `FEAT`: new validation logic for deployment diagram ([#1709](https://github.com/camunda/camunda-modeler/issues/1709))
```

Comment:
Don't we also ship a completely reworked deployment UX?

If that is the case, let's rename this one to completely overhauled deployment UX or something. Because this is what we did.

Reply:
+1 for this